### PR TITLE
[refactoring] Correct json/bytes conversion in queryItem(s)

### PR DESCRIPTION
**Pull request by Max-Pol**
@camillemarini @mgaltier 

I edited the returns value of query functions to return a json-formatted object.

### CODE REVIEW

File: orchestrator.go [hunk 1]
```diff
@@ -89,6 +90,9 @@ type Learnuplet struct {
 
 /*
  * The Init method is called when the Smart Contract orchestrator is instantiated by the blockchain network
+ * Note that chaincode upgrade also calls this function to reset
```
Comment (camillemarini) [inline orchestrator.go:92]: The init method is like this, for test purposes, waiting for Mattthieu to be done with updating the tests

**Review by camillemarini — APPROVED** [review]
Great, thanks for the fix

**Comment by camillemarini** [comment]
@mgaltier could you have a look to integrate it in your test, and if looks ok to you, merge it?

File: orchestrator.go [hunk 2]
```diff
@@ -380,20 +380,21 @@ func (s *SmartContract) queryItems(APIstub shim.ChaincodeStubInterface, args []s
...
-	if err != nil {
-		return shim.Error(err.Error())
+		payloadSlice = append(payloadSlice, queryResponse.GetValue())
 	}
 	fmt.Printf("- end looking for elements of type %s\n", objectType)
+
+	// Formatting in json
+	payload := append([]byte("["), bytes.Join(payloadSlice, []byte(","))...)
+	payload = append(payload, "]"...)
+
```
Comment (GuillaumeCisco) [inline orchestrator.go]: Did we not talk about this way on mattermost @camillemarini ?
Looks unmaintenable to me. Putting `[` and `]` looks hacky.

Comment (camillemarini) [reply orchestrator.go]: @GuillaumeCisco no, we didn't talk about this way on mattermost. It's a fix from Max-Pol to be able to parse output from the chaincode. Do you have an idea how to do it more properly?
Also @mgaltier is writting tests on the chaincode, and I think he is also looking at this

Comment (Max-Pol) [reply orchestrator.go]: And there is another issue btw: the "key" is not sent, only the "value".

Comment (camillemarini) [reply orchestrator.go]: @Max-Pol  When I call queryItems in the docker cli, I get the keys:
```
peer chaincode query -n mycc -c '{"Args":["queryItems", "algo"]}' -C $CHANNEL_NAME
```
returns:
```
Query Result: ["namespace:\"mycc\" key:\"algo_0\" value:\"{\\\"docType\\\":\\\"algo\\\",\\\"problem\\\":\\\"problem_1\\\"}\" ","namespace:\"mycc\" key:\"algo_1\" value:\"{\\\"docType\\\":\\\"algo\\\",\\\"problem\\\":\\\"problem_1\\\"}\" "]
```

Comment (GuillaumeCisco) [reply orchestrator.go]: I was thinking about https://mattermost.morpheo.co/morpheo/pl/c3ntdod5rfdsbju1sywi4a8ykr
Json stringify

Comment (Max-Pol) [reply orchestrator.go]: @camillemarini I meant "in the PR, the key is not sent".
@All I changed it, I believe there is no better way to do it. As you can see, it's a "vrai bonheur" to parse json in Go :)

**Pull request merged**